### PR TITLE
Add Ubuntu 22.04-arm64 for parallels on apple silicon

### PR DESCRIPTION
**Pull request by phillipross**
Signed-off-by: Phillip Ross <phillip.w.g.ross@gmail.com>

## Description
Add Ubuntu 22.04-arm64 box

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by vkarve-chef — APPROVED** [review]

**Comment by ashiqueps** [comment]
@Stromweld Me too doesn't have a arm machine to test the build.

File: packer_templates/ubuntu/ubuntu-22.04-arm64.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        "git_revision": "__unknown_git_revision__",
+        "guest_additions_url": "",
+        "headless": "",
+        "http_directory": "{{template_dir}}/http",
+        "http_proxy": "{{env `http_proxy`}}",
+        "https_proxy": "{{env `https_proxy`}}",
+        "hyperv_generation": "2",
+        "hyperv_switch": "bento",
+        "iso_checksum": "c209ab013280d3cd26a344def60b7b19fbb427de904ea285057d94ca6ac82dd5",
+        "iso_name": "ubuntu-22.04-live-server-arm64.iso",
```
Comment (svpernova09) [inline packer_templates/ubuntu/ubuntu-22.04-arm64.json:83]: This should be updated to `"iso_name": "ubuntu-22.04.1-live-server-arm64.iso",`.

```
┌─(~/Code/bento/packer_templates/ubuntu)(ubuntu-22.04-arm64 ??:2 M:1 S:14)
└─(130)─ packer build --only=parallels-iso ubuntu-22.04-arm64.json
parallels-iso: output will be in this color.

==> parallels-iso: Retrieving ISO
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04-live-server-arm64.iso
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04-live-server-arm64.iso?checksum=sha256%3Ac209ab013280d3cd26a344def60b7b19fbb427de904ea285057d94ca6ac82dd5
==> parallels-iso: Download failed bad response code: 404
==> parallels-iso: error downloading ISO: [bad response code: 404]
Build 'parallels-iso' errored after 5 seconds 905 milliseconds: error downloading ISO: [bad response code: 404]

==> Wait completed after 5 seconds 906 milliseconds

==> Some builds didn't complete successfully and had errors:
--> parallels-iso: error downloading ISO: [bad response code: 404]

==> Builds finished but no artifacts were created.
```

**Comment by svpernova09** [comment]
Once the URL & Checkup [are updated](https://github.com/chef/bento/pull/1423/files#r956340308) the box builds fine for me on a M1 2020 mac mini.  macOS 12.5.1 Parallels Version 18.0.0 (53049)

Build log: https://gist.github.com/svpernova09/31b987140d4c3176e6e9f307be1b9ee5

Vagrant Box: https://app.vagrantup.com/Svpernova09/boxes/demo-bento-22.04-ARM

File: packer_templates/ubuntu/ubuntu-22.04-arm64.json (hunk 1 above)
Comment (phillipross) [reply packer_templates/ubuntu/ubuntu-22.04-arm64.json]: Thanks! I've pushed an update to use the new iso version/checksum.

Comment (svpernova09) [reply packer_templates/ubuntu/ubuntu-22.04-arm64.json]: Thanks so much for this PR!

**Comment by svpernova09** [comment]
Changes looking good here. Should be safe to merge.

```
┌─(~/Code/bento/packer_templates/ubuntu)(ubuntu-22.04-arm64 ??:2 S:16)
└── packer build --only=parallels-iso --on-error=abort  ubuntu-22.04-arm64.json                                                                                                            ─(14:00:06)(08/27/22)─┘
parallels-iso: output will be in this color.

==> parallels-iso: Retrieving ISO
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.1-live-server-arm64.iso
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.1-live-server-arm64.iso?checksum=sha256%3Abc5a8015651c6f8699ab262d333375d3930b824f03d14ae51e551d89d9bb571c
==> parallels-iso: http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.1-live-server-arm64.iso?checksum=sha256%3Abc5a8015651c6f8699ab262d333375d3930b824f03d14ae51e551d89d9bb571c => /Users/halo/.cache/packer/3678bda67593f7a1fae1140298d3bcf1f2d45d0d.iso
==> parallels-iso: Starting HTTP server on port 8675
==> parallels-iso: Creating virtual machine...
==> parallels-iso: Applying default settings...
==> parallels-iso: Creating hard drive...
==> parallels-iso: Setting the boot order...
==> parallels-iso: Attaching ISO to the default CD/DVD ROM device...```

**Pull request merged**